jest.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  verbose: true,
```

Comment:
This defaults to false, so you seem to be changing the behavior of the test suite here in a way not required to do the work. I'd prefer not to see something like that rolled in without justification.

Reply:
This was actually copy paste job. I will investigate